codes.go

```diff
@@ -87,8 +99,12 @@ func movementCode(n uint, code rune) string {
 	return esc + strconv.FormatUint(uint64(n), 10) + string(code)
 }
 
-// Styler returns a func that applies the attributes given in the Styler call
-// to the provided string.
+// Styler is a closure that accepts multiple possible styling transforms from the state,
```

Comment:
`Styler` isn't a closure, as it's not defined within some other block that it can "close over".
It does no templating. It creates a new func that will apply the provided attributes to its argument.

Reply:
This made me realize I was using the word closure wrong, I meant a function that returns another function.